src/main/js/helpers/label.js

```diff
@@ -292,6 +295,25 @@ const translateLabelShapeContainer = (config, shapeContainerPath) => {
     }
 };
 
+/**
+ * Creates div element for popup tooltip
+ */
+const createTooltipDiv = () => {
+    d3.select("body")
+        .append("div")
+        .attr("id", styles.labelPopupTooltip)
+        .classed(styles.labelPopupTooltip, true);
+};
+
+/**
+ * Removes popup tooltip div element
+ */
+const destroyTooltipDiv = () => {
+    if (document.querySelector(`.${styles.labelPopupTooltip}`)) {
+        document.querySelector(`.${styles.labelPopupTooltip}`).remove();
+    }
```

Comment:
Do we need to remove `${styles.labelPopupOverlay}` also here?

Reply:
not necessary i guess veera because we are creating and removing it in same place if necessary.